app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_in = command.split(" ")[1]
```

Comment:
1. First of all you must check that command have exactly 3 parameters
2. Also space is the default argument in .split
3. You should read checklist:
https://github.com/Maksym-Filipchuk/py-copy-file/blob/develop/checklist.md#code-efficiency